content/docs/faq-state.md

```diff
@@ -6,6 +6,18 @@ layout: docs
 category: FAQ
 ---
```

Comment:
Adding a quick review here to get the ball rolling, sorry if it's lacking details:

- "What Should Go in State?" is a good way to frame this topic
- "What does setState do?" should remain at the top to give context to the other sections
- Might make sense to add "What is the difference between state and props" with a link to main docs on those topics, while explaining that both cause a re-render when they change. Some of the content below could be pulled into this section to focus more on the subjects of computed values and syncing props and state.

Reply:
Thanks for the review. Let me come up with a updated version and inform you. 
@alexkrolick